mango/logging/chrono.py

```diff
@@ -1,26 +1,26 @@
 import time
-import logging
-
-logger = logging.getLogger("root")
+from .logger import get_basic_logger
 
 
 class Chrono:
     """
     Class to measure time
     """
 
-    def __init__(self, name: str, silent: bool = False, precision: int = 2):
+    def __init__(self, name: str, silent: bool = False, precision: int = 2, logger=get_basic_logger()):
```

Comment:
This i think it should be better to be something like:

``` python 
class Chrono:
    logger = get_basic_logger()
    def __init__(self, name, silent=False, precision=2, logger=logger)

```

So we do not pass a function as an argument, even if the function always returns the logger object